app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -64,6 +64,8 @@ def requested_authn_context
     if AppConfig.env.aal_authn_context_enabled == 'true'
       requested_aal_authn_context
     else
+      sp_defined_aal_context = saml_request.requested_aal_authn_context
+      return sp_defined_aal_context if sp_defined_aal_context.presence
       requested_ial_authn_context
```

Comment:
I think something got lost in translation from my [previous suggestion](https://github.com/18F/identity-idp/pull/4698#discussion_r580696528)

I think this entire section can be replaced like this:

```suggestion
      sp_defined_aal_context.presence || requested_ial_authn_context
```

Reply:
Great suggestion. Thanks!